examples/campaign_management/add_campaign_labels.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example adds a campaign label to a list of campaigns.
+
+This example assumes that a label has already been prepared.
+"""
+
+import argparse
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+_DEFAULT_PAGE_SIZE = 1000
+
+
+def main(client, customer_id, label_id, campaign_ids, page_size):
+    """This code example adds a campaign label to a list of campaigns.
+
+    Args:
+        client: An initialized GoogleAdsClient instance.
+        customer_id: A client customer ID str.
+        label_id: A label ID to use when searching for campaigns.
+        campaign_ids: A list of campaign IDs to which the label will be added.
+        page_size: An int of the number of results to include in each page of
+            results.
+    """
+
+    # Get an instance of CampaignLabelService client.
+    campaign_label_service = client.get_service(
+        'CampaignLabelService', version='v3')
+
+    # Set the resource name of the label to be added across the campaigns.
+    label_resource_name = f'customers/{customer_id}/labels/{label_id}'
+
+    operations = []
+
+    for campaign_id in campaign_ids:
+        campaign_resource_name = f'customers/{customer_id}/'\
```

Comment:
ditto previous comment. Also don't use \ for continuation.

Reply:
Fixed.